src/server/slowlog.cc

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include "server/slowlog.h"
+
+namespace dfly {
+
+SlowLogEntry::SlowLogEntry(int64_t command_id, int64_t unix_timestamp, int64_t execution_time_micro,
+                           std::vector<std::string> cmd_args, std::string client_ip,
+                           std::string client_name) {
+  command_id_ = command_id;
+  unix_timestamp_ = unix_timestamp;
+  execution_time_micro_ = execution_time_micro;
+  cmd_args_ = cmd_args;
+  client_ip_ = client_ip;
+  client_name_ = client_name;
+}
+int64_t SlowLogEntry::CommandId() const {
+  return command_id_;
+}
+
+int64_t SlowLogEntry::UnixTimestamp() const {
+  return unix_timestamp_;
+}
+
+int64_t SlowLogEntry::ExecutionTimeMicro() const {
+  return execution_time_micro_;
+}
+
+std::string const& SlowLogEntry::ClientIp() const {
```

Comment:
I know this order works as well but we do not use it in the project,
please use `const T&` for constrefs